Sources/DeepLearning/Layer.swift

```diff
@@ -101,6 +101,80 @@ public extension Layer {
     }
 }
 
+extension Differentiable {
+
+    @differentiable(wrt: (self, l1, l2))
+    public func sequenced<L1: Layer, L2: Layer>(
+        in context: Context, through l1: L1, _ l2: L2)
+        -> L2.Output
```

Comment:
I was about to leave comments about indentation, but I thought "it's gonna be gyb'd anyway" :)

Reply:
Yeah, I don't have my editor set up to do the indentation automatically for me. :-( Sorry!